sdk/cosmos/microsoft-azure-cosmos/src/test/java/com/azure/data/cosmos/rx/TestSuiteBase.java

```diff
@@ -90,7 +90,7 @@
     protected static final int TIMEOUT = 40000;
     protected static final int FEED_TIMEOUT = 40000;
     protected static final int SETUP_TIMEOUT = 60000;
-    protected static final int SHUTDOWN_TIMEOUT = 12000;
+    protected static final int SHUTDOWN_TIMEOUT = 24000;
```

Comment:
I don't think we should increase timeouts
would be better to get approval from Cosmos team before doing it.

Reply:
> what's the reason the timeout value has changed? are you seeing some tests taking a long time?

The Cosmos build is/was failing. There were two issues that we were seeing. The first was an exception being thrown in afterClass for TestSuiteBase. I've filed an issue:

https://github.com/Azure/azure-sdk-for-java/issues/5215

Would be great if someone could investigate the failure. 12 seconds to delete something in an emulator seems pretty extraordinary, so 24 seconds is even worse - but it did seem to resolve that error.

What I'm trying to do with the ```java - cosmos``` build is get it to the point where it is reliably passing so that we can understand whether changes are having a negative impact or not. At this point the ```java - cosmos``` pipeline has never succeeded since it was created. We need to get it working before we can hope to release the package via our unified pipelines.